storageplugins/hcalendar/CalendarBackend.cpp

```diff
@@ -415,30 +415,30 @@ bool CalendarBackend::modifyIncidence( KCalendarCore::Incidence::Ptr aInci, cons
     KCalendarCore::Incidence::Ptr origInci = getIncidence ( aUID );
 
     if( !origInci ) {
-        LOG_WARNING("Item with UID" << aUID << "does not exist. Cannot modify");
+        qCWarning(lcSyncMLPlugin) << "Item with UID" << aUID << "does not exist. Cannot modify";
         return false;
     }
 
     if( !modifyIncidence( origInci, aInci ) ) {
-        LOG_WARNING( "Could not make modifications to incidence" );
+        qCWarning(lcSyncMLPlugin) << "Could not make modifications to incidence";
         return false;
     }
 
     if( commitNow )  {
         if( !iStorage->save() )
         {
-            LOG_WARNING( "Could not commit changes to calendar");
+            qCWarning(lcSyncMLPlugin) << "Could not commit changes to calendar";
             return false;
         }
-        LOG_DEBUG( "Single incidence committed");
+        qCDebug(lcSyncMLPlugin) << "Single incidence committed";
     }
 
     return true;
 }
 
 CalendarBackend::ErrorStatus CalendarBackend::deleteIncidence( const QString& aUID )
 {
-	FUNCTION_CALL_TRACE;
+	FUNCTION_CALL_TRACE(lcSyncMLPluginTrace);
```

Comment:
might as well fix the indentation while we're here, I suppose

Reply:
Done